core/lib/workarea/ext/jbuilder/jbuilder_cache.rb

```diff
@@ -0,0 +1,31 @@
+require 'jbuilder/jbuilder_template'
+
+decorate JbuilderTemplate do
```

Comment:
I had problems when just overriding the class because `JbuilderTemplate` has some absurd way it's getting loaded into the codebase. By making this a decorator, we're ensuring that it will get applied after the class has already been loaded and defined, so it will be possible to call `super` within it. Previously, I basically couldn't call `super` without it erroring.